xarray/core/dataarray.py

```diff
@@ -2490,15 +2490,19 @@ def to_masked_array(self, copy: bool = True) -> np.ma.MaskedArray:
     def to_netcdf(self, *args, **kwargs) -> Union[bytes, "Delayed", None]:
         """Write DataArray contents to a netCDF file.
 
-        All parameters are passed directly to `xarray.Dataset.to_netcdf`.
+        All parameters are passed directly to :py:meth:`xarray.Dataset.to_netcdf`.
 
         Notes
         -----
         Only xarray.Dataset objects can be written to netCDF files, so
         the xarray.DataArray is converted to a xarray.Dataset object
         containing a single variable. If the DataArray has no name, or if the
         name is the same as a co-ordinate name, then it is given the name
-        '__xarray_dataarray_variable__'.
+        "__xarray_dataarray_variable__".
```

Comment:
```suggestion
        ``"__xarray_dataarray_variable__"``.
```
or
```suggestion
        ``__xarray_dataarray_variable__``.
```
not sure which is better

Reply:
Voting for the second

Reply:
I went for  ` ``"__xarray_dataarray_variable__"`` ` to emphasize it's a string.